cmd/hw_02/main.go

```diff
@@ -0,0 +1,32 @@
+package main
+
+import (
+	"fmt"
+	"golang-course/cmd/hw_02/fibonacci/dp"
+	"golang-course/cmd/hw_02/fibonacci/recursive"
+)
+
+func main() {
+	fmt.Println("Enter n, amount of Fibonacci nums. Range: 1-93")
+	var n int
+	fmt.Scanf("%d\n", &n)
```

Comment:
You need to handle an error whenever you can. Yes, in this case, you can ignore the error, but next time it may not work.   Explicit error handling should become a habit.
```suggestion
	_, err := fmt.Scanf("%d\n", &n)
	if err != nil {
	    fmt.Println(err)
	    return
	}
```